docs/content/cheatsheet.md

```diff
@@ -0,0 +1,1159 @@
+
+
+See {ref}`glue/gluing` for more information.
+
+### Gluing Visualizations
+
+**Example**:
+``````md
+```{code-cell} ipython3
+from myst_nb import glue
+import matplotlib.pyplot as plt
+import numpy as np
+
+x = np.linspace(0, 10, 200)
+y = np.sin(x)
+fig, ax = plt.subplots()
+ax.plot(x, y, 'b-', linewidth=2)
+
+glue("glued_fig", fig, display=False)
+```
+
+This is an inline glue example of a figure: {glue:}`glued_fig`.
+This is an example of pasting a glued output as a block:
+```{glue:} glued_fig
+```
+``````
+
+**Result**:
+```{code-cell} ipython3
+from myst_nb import glue
+import matplotlib.pyplot as plt
+import numpy as np
+
+x = np.linspace(0, 10, 200)
+y = np.sin(x)
+fig, ax = plt.subplots()
+ax.plot(x, y, 'b-', linewidth=2)
+
+glue("glued_fig", fig, display=False)
+```
+
+This is an inline glue example of a figure: {glue:}`glued_fig`.
+This is an example of pasting a glued output as a block:
+```{glue:} glued_fig
+```
+
+See {ref}`glue/gluing` for more information.
+
+### Gluing Math
+
+**Example**:
+``````md
+```{code-cell} ipython3
+import sympy as sym
+x, y = sym.symbols('x y')
+z = sym.Function('z')
+z = sym.sqrt(x**2+y**2)
+glue("example_eq", z, display=False)
+```
+
+To glue a math equation try
+```{glue:math} example_eq
+:label: glue-eq-example
+```
+``````
+
+**Result**:
+```{code-cell} ipython3
+import sympy as sym
+x, y = sym.symbols('x y')
+z = sym.Function('z')
+z = sym.sqrt(x**2+y**2)
+glue("example_eq", z, display=False)
+```
+
+To glue a math equation try
+```{glue:math} example_eq
+:label: glue-eq-example
+```
+
+See {doc}`glue` for more information.
+
+## Reference Documents
+
+``````{list-table}
+:header-rows: 1
+:widths: 10 20 20
+
+* - Token
+  - Example
+  - Result
+* - ```md
+    {doc}`path/to/document`
+    ```
+  - ```md
+    See {doc}`../content/citations`
+    for more information.
+    ```
+  - See {doc}`../content/citations`
+    for more information.
+* - ```md
+    {doc}`text <path/to/document>`
+    ```
+  - ```md
+    See {doc}`here <../content/citations>`
+    for more information.
+    ```
+  - See {doc}`here <../content/citations>`
+    for more information.
+``````
+
+## Footnotes
+
+``````{margin}
+<br/><br/><br/><br/>
+```{note}
+Footnotes are displayed at the very bottom of the page.
+```
+``````
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 20 10
+
+* - Text
+  - Example
+  - Result
+* - ```md
+    [^ref]
+
+    [^ref]: Footnote text
+    ```
+  - ```md
+    This is an example of a footnote.[^footnote1]
+
+    [^footnote1]: The definition for referencing
+     footnotes is generally placed at the bottom
+     of the document.
+    ```
+  - This is a footnote reference.[^myref]
+``````
+
+[^myref]: This **is** the footnote definition.
+
+See [Footnotes](https://myst-parser.readthedocs.io/en/latest/using/syntax.html#syntax-footnotes) for more information.
+
+## Citations
+
+```{note}
+Make sure you have a reference bibtex file. You can create one by running `touch references.bib`.
```

Comment:
can we link to the references.bib file that's used by jupyter book (or some other bibtex file) for reference?

Reply:
Interestingly a bibtex file cannot be referenced with an inline `doc` role. Should I add a link to the Github `references.bib` file instead?

Reply:
try just using a `{download}` role instead of referencing it as a `{doc}`

Reply:
How's this:
``````
```{note}
Make sure you have a reference bibtex file. You can create one by running `touch references.bib`
or view our {download}`references.bib <../references.bib>` example.
```
``````

Reply:
looks good to me!

Reply:
resolved here 1be0c8a